src/scripts/scripts.js

```diff
@@ -0,0 +1,825 @@
+        enModified: 'V',
+        ruModified: 'М',
+      },
+      {
+        keyCode: 66,
+        en: 'b',
+        ru: 'и',
+        enModified: 'B',
+        ruModified: 'И',
+      },
+      {
+        keyCode: 78,
+        en: 'n',
+        ru: 'т',
+        enModified: 'N',
+        ruModified: 'Т',
+      },
+      {
+        keyCode: 77,
+        en: 'm',
+        ru: 'ь',
+        enModified: 'M',
+        ruModified: 'Ь',
+      },
+      {
+        keyCode: 188,
+        en: ',',
+        ru: 'б',
+        enModified: '<',
+        ruModified: 'Б',
+      },
+      {
+        keyCode: 190,
+        en: '.',
+        ru: 'ю',
+        enModified: '>',
+        ruModified: 'Ю',
+      },
+      {
+        keyCode: 191,
+        en: '/',
+        ru: '.',
+        enModified: '?',
+        ruModified: ',',
+      },
+      {
+        keyCode: 16,
+        en: 'shift',
+        ru: 'shift',
+        code: 'ShiftRight',
+      },
+      {
+        keyCode: 38,
+        en: 'up',
+        ru: 'up',
+        code: 'ArrowUp',
+      },
+      {
+        keyCode: 17,
+        en: 'ctrl',
+        ru: 'ctrl',
+        code: 'ControlLeft',
+      },
+      {
+        keyCode: 17,
+        en: 'win',
+        ru: 'win',
+        code: 'MetaLeft',
+      },
+      {
+        keyCode: 18,
+        en: 'alt',
+        ru: 'alt',
+        code: 'AltLeft',
+      },
+      {
+        keyCode: 32,
+        en: 'space',
+        ru: 'space',
+        code: 'Space',
+      },
+      {
+        keyCode: 18,
+        en: 'alt',
+        ru: 'alt',
+        code: 'AltRight',
+      },
+      {
+        keyCode: 17,
+        en: 'ctrl',
+        ru: 'ctrl',
+        code: 'ControlRight',
+      },
+      {
+        keyCode: 37,
+        en: 'left',
+        ru: 'left',
+        code: 'ArrowLeft',
+      },
+      {
+        keyCode: 40,
+        en: 'down',
+        ru: 'down',
+        code: 'ArrowDown',
+      },
+      {
+        keyCode: 39,
+        en: 'right',
+        ru: 'right',
+        code: 'ArrowRight',
+      },
+    ],
+  },
+
+  init() {
+    this.elements.main = document.createElement('div');
+    this.elements.textarea = document.createElement('textarea');
+
+    this.elements.textarea.classList.add('keyboard-output');
+    this.elements.main.classList.add('keyboard');
+
+    if (localStorage.getItem('language')) {
+      this.properties.language = localStorage.getItem('language');
+    }
+
+    this.elements.main.appendChild(this.createKeyboard(this.properties.language));
+    this.elements.keys = this.elements.main.querySelectorAll('.keyboard__key');
+
+    document.body.appendChild(this.elements.textarea);
+    document.body.appendChild(this.elements.main);
+  },
+
+  createKeyboard(language) {
+    const fragment = document.createDocumentFragment();
+
+    this.properties.keyLayout.forEach((key, i) => {
+      const keyEl = document.createElement('button');
+      const insertBrAfter = [8, 220, 13, 38].indexOf(key.keyCode) !== -1;
```

Comment:
It is difficult to understand what this code is for. It is better to use named variables or configs